src/main/resources/component-biz.conf

```diff
@@ -130,6 +130,25 @@ sirius.frameworks {
 
 }
 
+scripting {
+    sandbox {
+        rules {
+            "sirius.db.mixing.BaseEntity.getId" = true
+            "sirius.db.mixing.BaseEntity.getIdAsString" = true
+            "sirius.db.mixing.BaseEntity.getUniqueName" = true
+            "sirius.db.mixing.BaseEntity.isNew" = true
+            "sirius.db.mixing.types.BaseEntityRef.fetchValue" = true
+            "sirius.db.mixing.types.BaseEntityRef.getId" = true
+            "sirius.db.mixing.types.BaseEntityRef.getIdAsString" = true
+            "sirius.db.mixing.types.BaseEntityRef.is" = true
+            "sirius.db.mixing.types.BaseEntityRef.isFilled" = true
+            "sirius.db.mixing.types.SafeList.contains" = true
+            "sirius.db.mixing.types.SafeList.isEmpty" = true
+            "sirius.db.mixing.types.SafeList.isFilled" = true
```

Comment:
Why arent these done via annotations in db?

Reply:
Because sirius-db doesn't have sirius-web as dependency

Reply:
Yeah, that's a bit ugly... 
If Sacha's approach with provided dependencies do not work, reactivate the _OneSirius_ conversation? 😇